staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -125,9 +127,10 @@ func newAuthenticator(c *cache, config *api.ExecConfig) (*Authenticator, error)
 	}
 
 	a := &Authenticator{
-		cmd:   config.Command,
-		args:  config.Args,
-		group: gv,
+		cmd:         config.Command,
+		args:        config.Args,
+		group:       gv,
+		installHint: config.InstallHint,
```

Comment:
Preemptively clean the string?

```suggestion
		installHint: cleanANSIEscapeCodes(config.InstallHint),
```

Reply:
Love it. Done.